README.md

```diff
@@ -71,9 +68,9 @@ case class CaseClassA(
   fieldGpb: CaseClassB,
   fieldGpbOption: Option[CaseClassB],
   fieldGpbOptionEmpty: Option[CaseClassB],
-  fieldStringsList: immutable.Seq[String],
-  fieldOptionIntegersList: Option[List[Int]],
-  fieldOptionIntegersEmptyList: Option[List[Int]]
+  fieldStrings: immutable.Seq[String],
+  fieldOptionIntegers: Option[List[Int]],
```

Comment:
OK, I understand better now. It kind of makes sense however in my opinion it is not clear because type as `Option[List[_]]` are not common in Scala (usually empty list is used as `None` in this case) and it just makes the library unnecessarily more complex - it's trying to do more than it needs to. I would vote for removing this feature. You are not offering full validation of the GPB anyway... you still have to check whether the values makes sense and emptiness of a list is one of them IMHO.